test/infrastructure/docker/api/v1beta1/dockercluster_types.go

```diff
@@ -95,6 +95,21 @@ type DockerClusterStatus struct {
 	// Conditions defines current service state of the DockerCluster.
 	// +optional
 	Conditions clusterv1.Conditions `json:"conditions,omitempty"`
+
+	// v1beta2 groups all the fields that will be added or modified in MachineSet's status with the DockerCluster version.
```

Comment:
```suggestion
	// v1beta2 groups all the fields that will be added or modified in DockerCluster's status with the V1Beta2 version.
```

Let's check for other occurences of this copy&paste error

Reply:
Fixed all around